jws/jws.go

```diff
@@ -85,6 +85,14 @@ func DetachedPayload(detached bool) SignOpts {
 	}
 }
 
+// Purpose is an option that can be passed to [github.com/tbd54566975/web5-go/jws.Sign].
+// It is used to select the appropriate key to sign with
+func TYP(typ string) SignOpts {
```

Comment:
Check me here, should we go with `Type()` instead? (or worse yet, `Typ()`?)

Reply:
i personally feel that `Type` reads better.

Reply:
Agreed, done